File: desarrollo/Backend/PruebaPlayaSenator/PruebaPlayaSenator.Application/Dto/HotelDto.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace PruebaPlayaSenator.Application.Dto
{
    public class HotelDto
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public string Description { get; set; }
        public int IdCategory { get; set; }
        public int IdCity { get; set; }
        public int IdRelevance { get; set; }
        public string ShortImageTitle { get; set; }
        public byte[] ShortImageData { get; set; }
        public string LargeImageTitle { get; set; }
        public byte[] LargeImageData { get; set; }
    }
}

File: desarrollo/FrontEnd/src/app/entities/HotelDto.ts
import { CharacteristicDto } from "./CharacteristicDto";

export interface HotelDto {
    id: number;
    name: string;
    description: string;
    idCategory: number;
    categoryName: string;
    idCity: number;
    cityName: string;
    idRelevance: number; // Destacado u otros
    relevanceName: string;
    shortImageTitle: string;
    shortImageData: string;
    largeImageTitle: string;
    largeImageData: string;     

    listaCaracteristicas:CharacteristicDto[];
  }